### ISSUE TITLE
Mobile app can't connect after last update of add-on

### ISSUE THREAD

**Issue by Awo65** [hassio-addons/addon-zerotier#169]
# Problem/Motivation

> (Why the issue was filed)

## Expected behavior

> (What you expected to happen)

## Actual behavior

> (What actually happened)

## Steps to reproduce

> (How can someone else make/see it happen)

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

**Comment by Awo65** [comment]
lost connection via ZeroTier and mobile app. The last time I was connecting, was on Wednesday 08/03 - with no issue. On Monday 13/03 I tried with no success. On ZeroTier panel both nodes are online, but mobile app can’t connect. I spotted that in the meanwhile was update of this add-on.
I restarted system several times, installed all system updates - no result. 
In the log there is:

[18:34:10] INFO: Starting ZeroTier One…
s6-rc: info: service legacy-services successfully started

After some time In log appeared:

[18:34:10] INFO: Starting ZeroTier One…
s6-rc: info: service legacy-services successfully started
connect: Host is unreachable
connect: Host is unreachable
connect: Host is unreachable
connect: Host is unreachable
connect: Host is unreachable
connect: Host is unreachable
